infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -23,6 +23,60 @@ nightly_builds = [
 
 # Built on push to specific tag.
 versioned_builds = [
+  # Remove libtpu from PyPI builds
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
```

Comment:
nit: can you add `python_version = "3.8"` here for completeness?

Reply:
Sounds good, let me explicitly specify python version when we want to use the default 3.8. Added to the cuda build below as well